src/App/bellNotificationHelpers.js

```diff
@@ -1,6 +1,7 @@
 import axios from '../library/axiosRetry'
```

Comment:
Most of this diff is just adjusting notifications to operate on the notifications array instead of the `response.data` object that the API returns (which contains the notifications array on its `results` property). This is so we can use a utility function previously developed to get all the API 'pages' of data without rewriting the logic for it.